corehq/apps/linked_domain/README.md

```diff
@@ -30,3 +30,24 @@ DomainLink.link_domains('linked_domain_name', 'master_domain_name', remote_detai
 On remote HQ, enable `linked_domains` feature flag and navigate to `project settings > Linked Projects` page which has a UI to pull changes from master domain for custom data fields for Location, User and Product models, user roles and feature flags/previews.
 
 Linked apps can be setup between linked domains by running `link_app_to_remote` command on linked domain.
+
+# Linked Applications
+
+Linked applications predate linked domains and can be used independently of linked domains.
```

Comment:
Not true. If you link and app the linked domain record is automatically created.

Reply:
thanks, fixed